app/src/main/java/org/android/go/sopt/adapter/RepoTitleRVAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package org.android.go.sopt.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.android.go.sopt.databinding.ItemTitleBinding
+
+class RepoTitleRVAdapter(context: Context) : RecyclerView.Adapter<RepoTitleRVAdapter.RepoTitleViewHolder>() {
+
+    private val inflater by lazy { LayoutInflater.from(context) }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RepoTitleViewHolder {
+        return  RepoTitleViewHolder(ItemTitleBinding.inflate(inflater,parent,false))
+    }
+
+    override fun onBindViewHolder(holder: RepoTitleViewHolder, position: Int) {
+
+    }
+
+    override fun getItemCount(): Int {
+        return 1
+    }
```

Comment:
```suggestion
    override fun getItemCount(): Int = 1
```
코드가 한 줄이면 이렇게 쓰는 것도 좋습니다!